projects/wave-core-new/src/lib/project/project.service.ts

```diff
@@ -203,44 +150,44 @@ export class ProjectService {
     /**
      * Set the time of the current project.
      */
-    setTime(time: Time) {
-        // TODO: server communication
-        // this.project$.pipe(first()).subscribe(project => {
-        //     const oldTime = project.time;
-        //     if (time && time.isValid() && !time.isSame(oldTime)) {
-        //         this.changeProjectConfig({
-        //             time,
-        //         });
-        //     }
-        // });
+    setTime(time: Time): Observable<void> {
+        const subject = new Subject<void>();
```

Comment:
whta happens to this subject? is it stored somewhere?

Reply:
The idea is to make the call, no matter if the caller of `setTime` subscribes to the observable. This is where the subject comes in, which provides you the option to observe it.